sentence-transformers.nodeps.txt

```diff
@@ -0,0 +1,7 @@
+# These can be installed with --no-deps.
```

Comment:
hm, is this to separate out dependencies before we add other mechanisms?

Reply:
I did this because there was some concern about dependencies, but I'll admit this probably is not the best way to deal with it.  I'd actually recommend I remove this and add extras handling like caikit in a separate PR.  Is that preferred?  Or is it better to just accept the dependencies.